### ISSUE TITLE
`URLSearchParams` vs `URL/searchParams` discrepancy in Safari

### ISSUE THREAD

**Issue by jakub-g** [mdn/browser-compat-data#11330]
# TL;DR

If someone has access to Safari 10.0, could you verify the following:
```
typeof URLSearchParams
'searchParams' in URL.prototype
```

# Context

I found a discrepancy in two tightly lined APIs in Safari. To test it, you'd need Safari 10.0 which unfortunately I don't have (BrowserStack only has Safari 10.1)

# Good info ✅

https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams 
says Safari 10.1

This is confirmed here in TP14 release notes https://webkit.org/blog/6969/release-notes-for-safari-technology-preview-release-14/
and the blog says that 10.1 is TP14-TP20 https://webkit.org/blog/7477/new-web-features-in-safari-10-1/
so this seems correct.

# Questionable info 🤔 

However this:
https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams
says Safari 10.0

That would be strange. If `URLSearchParams` interface is not supported, what would `URL.searchParams()` return?
Perhaps it returns a generic object (this wouldn't be unheard of), but I find this suspicious

**Comment by jakub-g** [comment]
Note: the tricky thing is that mobile safari always says it's 10.0 even if it uses webkit/603.x The "real" 10.0 uses webkit/602.x. Best to check "applewebkit" part of `navigator.userAgent`.
